lib/kitchen/provisioner/chef_base.rb

```diff
@@ -290,6 +297,14 @@ def install_script_contents
           installer.install_command
         end
       end
+
+      def installer_script_name
+        "installer.#{powershell_shell? ? "ps1" : "sh"}"
+      end
+
+      def installer_script_path
```

Comment:
I would rename this to make it clear that this is the installer_script_path on the VM.